pootle/apps/pootle_project/models.py

```diff
@@ -402,12 +402,16 @@ def get_cachekey(self):
     # # # /TreeItem
 
     def get_stats_for_user(self, user):
-        self.set_children(self.get_children_for_user(user))
+        self.set_children(
+            self.get_children_for_user(
+                user, select_related=["language"]))
```

Comment:
this select related should come from views. language is not necessary for stats.